src/abstractitensornetwork.jl

```diff
@@ -623,18 +623,44 @@ function gauge_walk(
   return gauge_walk(alg, tn, edgetype(tn).(edges); kwargs...)
 end
 
+function tree_gauge(alg::Algorithm, ψ::AbstractITensorNetwork, region)
+  return tree_gauge(alg, ψ, [region])
+end
+
+#Get the path that moves the gauge from a to b on a tree
+#TODO: Move to NamedGraphs
+function gauge_path(g::AbstractGraph, region_a::Vector, region_b::Vector)
```

Comment:
Can we come up with a name that is more general that `gauge_path` for this, since now it isn't really specific to gauging? Maybe `trail_between_regions`?

Also, is this really a "path" by the definition in https://en.wikipedia.org/wiki/Path_(graph_theory)? It looks like in general this could be a "trail" since it is made from a spanning tree.

Reply:
I think it may not even be a walk actually as the edges may not be contiguous (for instance it could be like `[3 => 2 , 1 => 2]` to move the gauge center from `[1,2,3]` to `[2]` on a 4-site path). Maybe `edge_sequence_between_regions` ?

Reply:
Sounds good to me.